### PR TITLE
Dataflow: Remove accidentally exposed predicates.

### PR DESCRIPTION
**Pull request by aschackmull**
We should really only expose `PathGraph` directly from the module instantiation.

### CODE REVIEW

**Review by MathiasVP — COMMENTED** [review]
I guess _technically_ this does warrant a change note?

**Comment by aschackmull** [comment]
> I guess _technically_ this does warrant a change note?

Yes. Now to choose between `fix` and `breaking`.

File: cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl1.qll [hunk 1]
```diff
@@ -379,6 +378,8 @@ class PathNode instanceof I::PathNode {
   final predicate isSinkGroup(string group) { super.isSinkGroup(group) }
 }
 
+module PathGraph = I::PathGraph;
```
Comment (hvitved) [inline cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl1.qll:381]: Shouldn't we instead be redefining the `nodes`, `edges`, and `subpaths` predicates in terms of the `PathNode` type defined above?

Comment (aschackmull) [reply cpp/ql/lib/semmle/code/cpp/dataflow/internal/DataFlowImpl1.qll:381]: We could. Can you think of a case where it matters?

**Review by MathiasVP — APPROVED** [review]

**Pull request merged**